File: c/Linear_Search.c
#include<stdio.h>

int main()
{
	int a[]={2,4,632,53,7,55,62,12,32};
	int found=0;
	int data,i;
	int size=sizeof(a);
	
	printf("Enter the data you want to search in Linear Search:\n");
	scanf("%d",&data);
	
	for(i=0;i<size;i++)
	{
		if(a[i]==data)
		{
			printf("The required element is found at position %d",i++);
			found++;
			break;
		}
	}
	if(found==0)
	{
		printf("Element not present in given array");
	}
	return 0;
}

File: PYTHON/Linear_search.py
def search(arr, size, item):

	for i in range(0, size):
		if (arr[i] == item):
			return i
	return -1

if __name__ == "__main__":
	arr = [2, 3, 4, 8, 13, 24, 10, 40]
	item = 13
	size = len(arr)

	result = search(arr, size, item)
	if(result == -1):
		print("Element is not present in array")
	else:
		print("Element is present at index", result)
